www/_posts/2017-12-04-cordova-android-7.0.0.md

```diff
@@ -0,0 +1,58 @@
+
+To upgrade:
+
+    npm install -g cordova
+    cd my_project
+    cordova platform remove android
+    cordova platform add android@7.0.0
+
+To add it explicitly:
+
+    cordova platform add android@7.0.0
+
+<!--more-->
+## Curated Changelog
+* [CB-13612](https://issues.apache.org/jira/browse/CB-13612) Fix the remapper so that XML files copy over and the Camera works again.
+* [CB-13741](https://issues.apache.org/jira/browse/CB-13741) Bump `package.json` so we can install plugins
+* [CB-13610](https://issues.apache.org/jira/browse/CB-13610) Compress the default app assets
+* [CB-12835](https://issues.apache.org/jira/browse/CB-12835) add a Context getter in CordovaInterface
+* [CB-8976](https://issues.apache.org/jira/browse/CB-8976) Added the `cdvVersionCodeForceAbiDigit` flag to the template build.gradle that appends 0 to the versionCode when `cdvBuildMultipleApks` is not set
+* [CB-12291](https://issues.apache.org/jira/browse/CB-12291) (android) Add x86_64, arm64 and armeabi architecture flavors
+* [CB-13602](https://issues.apache.org/jira/browse/CB-13602) We were setting the path wrong, this is hacky but it works
+* [CB-13601](https://issues.apache.org/jira/browse/CB-13601) Fixing the standalone run scripts to make sure this works without using the CLI
+* [CB-13580](https://issues.apache.org/jira/browse/CB-13580) fix build for multiple apks (different product flavors)
+* [CB-13558](https://issues.apache.org/jira/browse/CB-13558) Upgrading the gradle so we can upload the AAR
+* [CB-13297](https://issues.apache.org/jira/browse/CB-13297) This just works once you bump the project structure.  Java 1.8 compatibility baked-in
+* [CB-11244](https://issues.apache.org/jira/browse/CB-11244) **Android** Studio 3 work, things have changed with how the platform is built
+* [CB-11244](https://issues.apache.org/jira/browse/CB-11244) Found bug where the gradle subproject changes weren't actually getting written to the correct gradle file
+* [CB-13470](https://issues.apache.org/jira/browse/CB-13470) Fix Clean so that it cleans the **Android** Studio structure
+* [CB-11244](https://issues.apache.org/jira/browse/CB-11244) Adding specs for resource files inside an **Android** Studio Project
+* [CB-11244](https://issues.apache.org/jira/browse/CB-11244) Added remapping for drawables
+* [CB-11244](https://issues.apache.org/jira/browse/CB-11244) Found bug in Api.js where xml/strings.xml is used instead of values/strings.xml
+* [CB-11244](https://issues.apache.org/jira/browse/CB-11244) Setup Api.js to support multiple builders based on project structure
+* [CB-11244](https://issues.apache.org/jira/browse/CB-11244) Changing directory creation, will most likely hide this behind a flag for the next release of `cordova-android`, and then make it default in the next major pending feedback
+* Adding the Studio Builder to build a project based on **Android** Studio, and deleting Ant, since Google does not support Ant Builds anymore. Sorry guys!
```

Comment:
Sorry guys => Sorry folks

Reply:
You are aware that this is taken directly from the commit messages, right?

Reply:
No, as it says "Curated Changelog" as headline above.

The commit message was similarly problematic then. No need to keep this in the blog post though.

Reply:
@janpio The commit is in the changelog because dropping ANT support is important.  I've edited it.  It was an innocent mistake that I can't fix without violating a bunch of ASF policies.

Reply:
Removing the word - as you did - is totally fine, thank you.